test/unit/contextResponses/contextResponseNumericWithFixedTimeInstantUpdate.json

```diff
@@ -0,0 +1,29 @@
+{
+  "contextResponses": [
+    {
+      "contextElement": {
+        "attributes": [
+          {
+            "name": "attrNameNumeric",
+            "type": "attrType",
+            "value": "222",
+            "metadatas": [
+              {
+                "name": "TimeInstant",
+                "type": "ISO8601",
+                "value": "1970-01-01T00:00:00.000"
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-sth-comet/commit/2cdb0828b9cb7692585a2f0fff83dec156481497